src/components/ChooseSubmodule/SearchModal.tsx

```diff
@@ -0,0 +1,94 @@
+import {
```

Comment:
This is just the old `SearchPopover.tsx` except `Popover` became `Modal` across the board, and the `ModalContent` needed to be wrapped in a `ModalOverlay`